packages/extension-browser/webpack.config.js

```diff
@@ -9,6 +9,11 @@ const baseConfig = {
                 test: /axe-core/,
                 use: 'raw-loader'
             },
+            // Bundle `js-library-detector as a raw string so it can be injected at runtime.
+            {
+                test: /js-library-detector/,
+                use: 'raw-loader'
+            },
```

Comment:
This will go away if we use `require('./snyk-snapshot.json')`

Reply:
I'm not sure about this, `js-library-detector` is used to generate the script to run inside the browser.

Reply:
Yep, got overzealous on my comments. This one would still be needed. ;)